### DISCUSSION TITLE
Download Artwork, Widescreen Cheats and Game settings for OPL

### DISCUSSION THREAD

**Discussion by linuxxp** [israpps/HDL-Batch-installer#67] [Q&A] [answered]
Hello,
How is possible to do ART and CFG download to HDD with HDL Batch installer? I can't find such option in menu.
Thank you!

**Comment by israpps** [comment, marked as answer]
> Hello, How is possible to do ART and CFG download to HDD with HDL Batch installer? I can't find such option in menu. Thank you!

Mark the games, right click, then chose the download files

**Reply by linuxxp** [reply]
Thank you!
